ts/screens/messages/paginated/MessagesHomeScreen.tsx

```diff
@@ -166,12 +172,12 @@ const MessagesHomeScreen = ({
               <SearchNoResultMessage errorType="InvalidSearchBarText" />
             ) : (
               <MessagesSearch
-                messages={[]}
+                messages={searchMessages}
                 searchText={_}
                 renderSearchResults={results => (
                   // TODO: filter may happen down the line
```

Comment:
⛏️ **nitpick**: we can remove this comment now 🎉

Reply:
Fixed with 9bd1ce1d.